hr_employee_cost_history/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
issue(legal): This is wrong, here and in all other files: 
```suggestion
# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
```

Reply:
@yajo also permission issues this demo user are also solved.